README.md

```diff
@@ -16,6 +16,8 @@ To use the library, include in your toml:
 stellar-strkey = "..."
 ```
 
+By default, the library uses the standard library (std). If you want to use it in a `no_std` environment, you need to disable the `default` features and enable the `alloc` crate.
```

Comment:
I don't think users will need to specifically enable the alloc crate? The fact that this crate extern references the alloc crate makes it required. Am I missing something?
```suggestion
By default, the library uses the standard library (std). If you want to use it in a `no_std` environment, you need to disable the `default` features.
```

Reply:
Yes, there is an issue with the wording here. I want to mention that this library uses `alloc` because when using this library in a `no_std` environment, due to the absence of a default global memory allocator, users need to add one themselves.

Perhaps it would be better to modify the expression as follows?

> This crate can be used in `no_std` environments. However, please note that it relies on the `alloc` crate for certain types such as `Vec`. When using this library in a `no_std` context, you are required to provide a global memory allocator. You can define a global allocator using the [`#[global_allocator]`](https://doc.rust-lang.org/std/alloc/index.html#the-global_allocator-attribute) attribute.

Reply:
I like it.

I think it would also be reasonable to link out to instructions for how to use the `alloc` crate, instead of providing them here. For example, this page shows how to use the alloc crate: https://docs.rust-embedded.org/book/collections/

But I don't feel strongly one way or the other. 👍🏻